cmd/minishift/cmd/openshift/registry.go

```diff
@@ -29,8 +29,8 @@ import (
 
 var registryCmd = &cobra.Command{
 	Use:   "registry [flags]",
-	Short: "Prints host and port of the OpenShift registry.",
-	Long:  `Prints host and port of the OpenShift docker registry in the format 'host:port'.`,
+	Short: "Prints the host name and port number of the OpenShift registry to the console.",
```

Comment:
"to standard output". This is also what we use in for example the `ip` command. The output goes to standard output which per default will be the console.